kernel/src/main/java/org/kframework/compile/checks/CheckAtt.java

```diff
@@ -71,85 +103,109 @@ private void checkLabel(Sentence sentence) {
       if (label.contains("`") || whitespace.matcher(label).find()) {
         errors.add(
             KEMException.compilerError(
-                "Rule label '" + label + "' cannot contain whitespace or backticks.", sentence));
+                "Label '" + label + "' cannot contain whitespace or backticks.", sentence));
```

Comment:
According to `Att.scala`, the `label` attribute can appear on any sentence, so made the message here more general